Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -62,7 +62,16 @@ moduleConfigProperty :
 ;
 
 moduleConfigElement :
-    unrestrictedIdentifier whiteSpace? EQ whiteSpace? expression (COLON numberLiteral)? endOfStatement
+    (unrestrictedIdentifier | lExpression) whiteSpace? EQ whiteSpace? (shortcut | resource | expression) endOfStatement
```

Comment:
I've just noticed that the final `| expression` is kinda redundant here, as it's captured by the resource rule. But it does no functional harm and helps to describe intent, so should it remain?